Probesys/bileto
# imp: Forbid actions in closed tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/275

## How to test manually



1. Close a ticket
2. Verify that all the actions are disabled / not available except reopening the ticket
3. Reopen the ticket
4. Verify the ticket is reopened

## Checklist
